implicit.cabal

```diff
@@ -43,6 +43,7 @@ Library
         -rtsopts
         -funfolding-use-threshold=16 
         -fspec-constr-count=10
+        -fno-warn-tabs
```

Comment:
Lol... since 7.10, ghc disagrees with you on tabs, but I'm not in a position to. :smile:

Nice work on the rest, getting it up to speed with the AMP changes.